pkg/blobfuse-proxy/init.sh

```diff
@@ -89,4 +90,10 @@ then
   sed -i 's/PRUNEFS="NFS/PRUNEFS="fuse blobfuse NFS/g' ${updateDBConfigPath}
   echo "after change:"
   cat ${updateDBConfigPath}
-fi
\ No newline at end of file
+fi
+
+AWK_PATH=$(which awk)
```

Comment:
add a new flag to control whether set read-ahead for nfs mount, by default is true